blurb/blurb.py

```diff
@@ -874,11 +892,12 @@ def add():
 Add a blurb (a Misc/NEWS entry) to the current CPython repo.
     """
 
-    editor = find_editor()
+    editor_command = find_editor()
 
     handle, tmp_path = tempfile.mkstemp(".rst")
     os.close(handle)
     atexit.register(lambda : os.unlink(tmp_path))
+    editor_command.append(tmp_path)
```

Comment:
editor_command is used in line 837. I don't see the append you mention. I'll need to rebase and test this again, lot of time has passed since I sent this :-)